doc/source/license.rst

```diff
@@ -2,4 +2,55 @@
 
 .. index:: license
 
-.. include:: ../../LICENSE.rst
```

Comment:
Not using a relative path for LICENSE.rst since it's outside of `doc` when it is hosted via GitHub Pages